test/v1/controllers/AmendAnnualSubmissionControllerSpec.scala

```diff
@@ -45,12 +45,12 @@ class AmendAnnualSubmissionControllerSpec
   private val taxYear: String    = "2019-20"
 
   private val testHateoasLinks: Seq[Link] = Seq(
-    Link(
+    hateoas.Link(
```

Comment:
```suggestion
   Link(
```
This can be updated for all the other links.